src/lib/UI/Config/Provider/FieldType/RichText/AlloyEditor.php

```diff
@@ -44,4 +45,20 @@ protected function getExtraPlugins(): array
 
         return [];
     }
+
+    /**
+     * This will be deprecated in eZ Platform 3.x.
+     * The alternative and more flexible solution will be introduced.
+     * So you will need to update Online Editor Extra Buttons as part of eZ Platform 3.x upgrade.
+     *
+     * @return array Custom plugins
+     */
+    protected function getExtraButtons(): array
+    {
+        if (isset($this->alloyEditorConfiguration['extra_buttons'])) {
```

Comment:
No longer need to use `if(isset(...))` condition because there is shorthand notation:

```php
return $this->alloyEditorConfiguration['extra_buttons'] ?? [];
```